### PR TITLE
Hotfix/lints

### PR DESCRIPTION
**Pull request by abea**
Adds apostrophe linting.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -16,7 +15,7 @@ if (argv.data) {
   // that don't need a directory." But create it if it's
   // somehow missing (Mac for instance).
   if (!fs.existsSync('/var/lib/misc')) {
-    fs.mkdirSync('/var/lib/misc', 0700);
+    fs.mkdirSync('/var/lib/misc', 700);
```
Comment (boutell) [inline app.js]: This was an intentional use of an octal constant.

If we're going to change to the ES6 version then it needs to be:

0o700

Just changing it to 700 would give you a different thing.

I just did some testing and the "new" syntax works back to node 4 (looks like that's when it came in). That's plenty old enough so please do it that way, with the `0o` prefix.

File: lib/boring.js [hunk 2]
```diff
@@ -11,7 +11,7 @@
 module.exports = function() {
   var args = process.argv.slice(2);
   var result = {
-    _ = []
```
Comment (boutell) [inline lib/boring.js]: weird how did this ever work? property initialization syntax maybe

Comment (abea) [reply lib/boring.js]: I'm not sure we're even using this file anymore. boring is now its own package that you imported in another file.

Comment (boutell) [reply lib/boring.js]: Yeah we probably aren't.

**Comment by abea** [comment]
More opportunities to make "boring" jokes, please.

**Pull request merged**